extensions/wikia/ArticleComments/js/ArticleComments.js

```diff
@@ -11,7 +11,12 @@
 	}
 
 	var $window = $(window),
-		ArticleComments;
+		ArticleComments,
+		track;
+
+	function isReply(element) {
+		return $(element).parents('.sub-comments').length;
```

Comment:
I'm 99% sure that it won't be an issue, but this isn't really a boolean function what name might actually suggest.

Reply:
yep, makes sense but this code will die soon (at it's an old platform) so it doesn't have to be perfect ;)